### PR TITLE
Refer to regular setup method

### PR DESCRIPTION
**Pull request by gliptak**
Signed-off-by: Gábor Lipták <gliptak@gmail.com>

https://github.com/wesm/vbench/issues/26

Successfully installed with ```pip install git+git://github.com/gliptak/vbench@setup1```

### CODE REVIEW

**Comment by gliptak** [comment]
@wesm Please review. Thanks

File: setup.py [hunk 1]
```diff
@@ -1,7 +1,9 @@
 #/usr/bin/env python
 
-from numpy.distutils.misc_util import Configuration
-from numpy.distutils.core import setup
+try:
+        from setuptools import setup
+except ImportError:
+        from distutils.core import setup
```
Comment (wesm) [inline setup.py]: Can you use 4-space indentation?

Comment (gliptak) [reply setup.py]: updated

**Comment by wesm** [comment]
Thanks!

**Pull request merged**